### PR TITLE
[KOGITO-5063] BDD tests: Support using builder images from insecure r…

### PR DESCRIPTION
**Pull request by sutaakar**
…egistry

https://issues.redhat.com/browse/KOGITO-5063

Signed-off-by: Karel Suta <ksuta@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by radtriste — APPROVED** [review]
one minor comment

File: test/pkg/steps/kogitobuild.go [hunk 1]
```diff
@@ -117,3 +134,33 @@ func getKogitoBuildConfiguredStub(namespace, runtimeType, serviceName string, ta
 
 	return buildHolder, err
 }
+
+func makeImageStreamInsecure(namespace, insecureImageTag string) error {
+	// Need to wait as operator overrides image stream in initial reconciliation
+	time.Sleep(2 * time.Second)
```
[the remark this answers is not in the document]
Comment (sutaakar) [reply test/pkg/steps/kogitobuild.go:140]: good point

**Comment by sutaakar** [comment]
@radtriste removed the implementation from Kogito operator, will be applied just on product.

**Review by radtriste — APPROVED** [review]

**Pull request merged**